### PR TITLE
Added number validations + tests

### PR DESCRIPTION
**Pull request by rkrishnan8594**

### CODE REVIEW

File: addon/validators/number.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+import Ember from 'ember';
+import buildMessage from 'ember-changeset-validations/utils/validation-errors';
+
+const {
+  isPresent,
+  isEmpty
+} = Ember;
+
+function _isNumber(value) {
+  return typeof value === 'number' && !isNaN(value);
```
Comment (poteto) [inline addon/validators/number.js]: Use and destructure `Ember.typeOf` please

File: addon/validators/number.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    return buildMessage(key, 'even', opts);
+  }
+
+  return true;
+}
+
+export default function validateNumber(opts = {}) {
+  return (key, value) => {
+    let numValue = Number(value);
+    let optKeys = Object.keys(opts);
```
Comment (poteto) [inline addon/validators/number.js]: Can destructure Object.keys

**Comment by poteto** [comment]
Looks pretty good, can you also update the readme?

**Comment by rkrishnan8594** [comment]
Sure thing 👍

**Comment by poteto** [comment]
Thanks @rkrishnan8594! 👏

**Pull request merged**